structure/hash.go

```diff
@@ -33,6 +34,7 @@ func (t *TxStructure) HSet(key []byte, field []byte, value []byte) error {
 	if t.readWriter == nil {
 		return ErrWriteOnSnapshot
 	}
+	t.SetDiskFullOpt(kvrpcpb.DiskFullOpt_AllowedOnAlmostFull)
```

Comment:
same as I said before, manipulation of (structure is a element of meta)Meta can only be set in the initial phase of Meta

Reply:
all here